app/models/product.rb

```diff
@@ -1,5 +1,11 @@
 class Product < ApplicationRecord
 
+  enum kind: {
```

Comment:
what is the need to pass hash in this case? cant we just pass an array?

Reply:
Yes sir, array used.